hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -365,6 +349,38 @@ public static void interruptCurrentThread(final int delayMillis) {
         }).start();
     }
 
+    public static <T> boolean containsIn(T item1, Collection<T> collection, Comparator<T> comparator) {
+        for (T item2 : collection) {
+            if (comparator.compare(item1, item2) == 0) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Returns the cartesian product of the specified lists.
+     */
+    public static List<Object[]> parameters(List<?>... lists) {
+        return Lists.cartesianProduct(lists).stream().map(List::toArray).collect(toList());
+    }
+
+    /**
+     * Computes the cartesian product of the values of the specified maps and
+     * prepends each result array with the comma-separated list of corresponding keys.
+     */
+    @SafeVarargs
+    public static List<Object[]> parameters(Map<String, ?>... maps) {
+        return Sets.cartesianProduct(
+                        Arrays.stream(maps).map(Map::entrySet).collect(toList())
+                ).stream().map(pairs ->
+                        Stream.concat(
```

Comment:
```suggestion
new Object[2] { 
// same  joining not wrapped in Stream.of,
Util.toList(pairs, Entry::getValue)
}
```

toList() can be pulled from Jet's Util class (which can be not-only-Jet now)